source/js/foundation/inject-react/pulse-project-list.js

```diff
@@ -10,7 +10,8 @@ export default (apps, env) => {
   document.querySelectorAll(`.pulse-project-list`).forEach((target) => {
     apps.push(
       new Promise((resolve) => {
-        ReactDOM.render(
+        const root = createRoot(container);
```

Comment:
I have question about what `container` references to

Reply:
should be target!